README.md

```diff
@@ -35,6 +71,8 @@ which results in truncated labels, so make sure to embed fonts.
 For drawio a 
 [desktop installation](https://github.com/jgraph/drawio-desktop) is also available.
 
-The latest editor's draft is available [here](https://w3c.github.io/wot-discovery/).
+## Known Implementations
 
-The latest Implementation Report is available [here](https://w3c.github.io/wot-discovery/testing/report.html).
+The W3C WoT collects known implementations at <https://www.w3.org/WoT/developers/>. Implementations of Discovery are found under categories 
+"TD Directory (TDD) Implementations", "Runtime Implementations for TD Consumers" and "Runtime Implementations for TD Exposers". Further information
```

Comment:
I'm not sure how useful it is to mention the heading titles. Is anyone going to read these here and try to find them there? Will anyone update the heading titles here if they change over time?

IMO, the developers page should have better grouping or labeling to make it clear which of those implement the discovery spec.

Reply:
I found that pretty weird too, but I was trying to be as close as possible to the template to avoid inconsistencies. We should mention this point in wot main repository and update the template.